### PR TITLE
docs: add zkSBT SDK & manta extension user guide

### PR DESCRIPTION
**Pull request by jincan39**

### CODE REVIEW

File: docs/calamari/DevelopOnCalamari.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+## Faucet
+
+To use the Dolphin faucet, head to our [Discord channel](https://discord.com/channels/795390654628102165/1055864933692219453).
+
+To receive tokens from the faucet, you'll need to get the attention of the faucet bot. Start by typing the command for the token you want, such as `/gimme-dol`. Discord will display a small pop-up asking for your wallet address.
+
+Type or paste your wallet address into the address field, then press enter to trigger the faucet request.
+
+You can request each of the four supported tokens (`DOL`, `KAR`, `KSM`, `MOVR`) once per UTC day. If you request the same token twice in the same day, the faucet bot will only respond with the status of your first request of the day. You can check the status of your address at https://faucet.dolphin.community/.
```
Comment (ghzlatarev) [inline docs/calamari/DevelopOnCalamari.md:84]: can they check Calamari addresses here ?

File: docs/calamari/DevelopOnCalamari.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+cargo run --release -- --chain=calamari-localdev --alice --tmp
+```
+
+Note: `calamari-localdev` eliminates the need for a relay chain and creates blocks automatically when transactions are received.
+
+Note: For a complete simulation, you can run a relay chain with a full parachain using tools like [zombienet or polkadot-launch](#Development-Tools). In that case you need to run the node with `--chain=calamari-local`
+
+## Chain ID
+
+Dolphin testnet chain id is `2084`
```
[the remark this answers is not in the document]
Comment (jincan39) [reply docs/calamari/DevelopOnCalamari.md:66]: @ghzlatarev Do you know what's the chain id of Calamari testnet?

File: docs/calamari/DevelopOnCalamari.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
+You can request each of the four supported tokens (`DOL`, `KAR`, `KSM`, `MOVR`) once per UTC day. If you request the same token twice in the same day, the faucet bot will only respond with the status of your first request of the day. You can check the status of your address at https://faucet.dolphin.community/.
+
+## Development Tools
+
+* `manta.js Javascript SDK`: A set of functionality which connects your public address (polkadot-js), your private addresses (`manta-signer`) and manages the creation of the payloads required for `MantaPay` extrinsics.
+* `Manta-Signer`: The signer is a desktop application which generates zero knowledge proofs and manages the private keys of your UTXOs. The signer is open sourced and you find instruction to set it up locally at [manta-signer](https://github.com/Manta-Network/manta-signer).
+* [zombienet](https://github.com/paritytech/zombienet) or the deprecated [polkadot-launch](https://github.com/paritytech/polkadot-launch) tool can help you set up a full relaychain + parachain network.
+* `Manta zkSBT SDK`: zkSBT sdk is extension of manta-pay sdk, and add some custom logic, including:
```
[the remark this answers is not in the document]
Comment (jincan39) [reply docs/calamari/DevelopOnCalamari.md:91]: Yeah think it need to be discussed

[the remark this answers is not in the document]
Comment (jincan39) [reply docs/calamari/DevelopOnCalamari.md]: it's the superset of manta.js, so i think they are parallel.

**Comment by Garandor** [comment]
image
this sidebar structure looks needlessly repetititve.
Maybe change the file name to "Get Started" or smth like that

File: docs/calamari/DevelopOnDolphin.md [hunk 4]
```diff
@@ -36,30 +33,70 @@ cd Manta
...
-or:
+Alternatively, you can build and run in one step:
 
 ```sh
 cargo run --release -- --chain=calamari-localdev --alice --tmp
 ```
 
-Note: `calamari-localdev` eliminates the need for a relay chain and creates blocks automatically when transactions are received.
+:::note
+`calamari-localdev` exposes the websocket at port 9944 by default, if you need to customize this, you can pass an additional parameter `--ws-port 42069` to use e.g port 42069 instead.
```
Comment (Garandor) [inline docs/calamari/DevelopOnDolphin.md:69]: `dolphin-localdev` everywhere

File: docs/calamari/DevelopOnDolphin.md [hunk 5]
```diff
@@ -1,30 +1,27 @@
...
 ## Introduction
-This guide provides step-by-step instructions to connect to a Dolphin node to test your applications on the public testnet named Dolphin. Dolphin is the public testnet of Calamari, which in turn is Manta Network's canary. A Dolphin node acts as your personal development environment to build and test applications for the Calamari Network.
 
-## Download Binary File
+A blockchain app typically consists of a frontend application connecting to and interacting with a blockchain node via RPC or Websocket.
 
-To download an official release of Calamari, visit our [release page](https://github.com/Manta-Network/Manta/releases).
+This guide provides step-by-step instructions test to your frontend application by
+- creating a non-networked local blockchain node for no-stakes testing, or
+- connecting to a node on the live Calamari Network.
```
Comment (Garandor) [inline docs/calamari/DevelopOnDolphin.md:9]: calamari network. replace everywhere...

**Review by Garandor — CHANGES_REQUESTED** [review]
There are still some mentions of Calamari in the dolphin guide

**Review by Garandor — APPROVED** [review]

**Pull request merged**